src/project.jar

```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
```

Comment:
Usually it's a bad practice to commit generated files

Reply:
I removed it from source code